Generator/GeneratorRegistry.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Pim\Bundle\DataGeneratorBundle\Generator;
+
+/**
+ * Registry for Generators.
+ *
+ * @author    Pierre Allard <pierre.allard@akeneo.com>
+ * @copyright 2016 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class GeneratorRegistry implements GeneratorRegistryInterface
+{
+    /** @var GeneratorInterface[] */
+    protected $generators = [];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function register(GeneratorInterface $generator)
```

Comment:
there should be a position here
because sometimes, you can't generate Foo before having generated BAR